### DISCUSSION TITLE
Unicode bidi attacks

### DISCUSSION THREAD

**Discussion by cobbdb** [remarkjs/remark#904] [Q&A] [answered]
Hello, does react-markdown employ measures against unicode bidi attacks?

https://trojansource.codes/
https://github.com/lirantal/anti-trojan-source

Also asked in rehype-santize: https://github.com/rehypejs/rehype/discussions/80

**Comment by wooorm** [comment, marked as answer]
We follow CommonMark for markdown (and for HTML, the HTML standard). You can try and raise this with those specs perhaps. But, see the other discussion.
